projects/ad4630_fmc/common/ad463x_fmc.txt

```diff
@@ -0,0 +1,21 @@
+FMC_pin FMC_port    Schematic_name  System_top_name       IOSTANDARD Termination
+
+# ad463x
```

Comment:
Add an empty line after line 3, and remove the empty line at line 21

Reply:
Fixed!